### DISCUSSION TITLE
supabase function error

### DISCUSSION THREAD

**Discussion by Titus777** [supabase/supabase#29113] [Questions]
im getiing this error within python, how can I fix it? 

Failed to delete old data: 404, {"code":"PGRST202","details":"Searched for the function public.delete_old_data without parameters or with a single unnamed json/jsonb parameter, but no matches were found in the schema cache.","hint":null,"message":"Could not find the function public.delete_old_data without parameters in the schema cache"}

**Comment by GaryAustin1** [comment]
You should provide some information on the Postgrest function definition and the call you are using.
